spec/lib/models/revocable_spec.rb

```diff
@@ -11,6 +11,8 @@
 
   describe :revoke do
     it 'updates :revoked_at attribute with current time' do
+      skip 'No assertion.'
+
```

Comment:
This follows the setup / exercise / assert flow, I'd like most specs (if not all) to read like that.